components/raftstore/src/store/peer.rs

```diff
@@ -1221,7 +1221,7 @@ where
     pub fn destroy(
         &mut self,
         engines: &Engines<EK, ER>,
-        perf_context: &mut EK::PerfContext,
+        perf_context: &mut ER::PerfContext,
```

Comment:
But no trackers are passed when calling `perf_context.report_metrics`, will this observation take any effect? And I'm not sure if recording the perf context in destroy is required, is it frequently happening?

Reply:
`detroy` is not bounded to any user request, so it will not be recorded to the tracker.